src/components/accountView/__tests__/accountView.test.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { mount } from 'enzyme';
+import AccountView from '../accountView';
+
+describe('AccountView Component', () => {
+  it('should render', () => {
+    const primaryClick = jest.fn();
+    const props = {
+      primaryClick
+    };
+
+    const component = mount(<AccountView {...props} />);
+    expect(component.render()).toMatchSnapshot();
```

Comment:
This one is curious... agree on using the "toJson" aspect, works great when combined with view level components, clears the clutter. In this case however, a stateless component, the "render" appears more helpful since the aim is detecting changes in output 

May have to revisit the concept for stateless, or focus on assertions... though as is, breezing over assertions on fine grain detail while still being altered to output changes is appealing (possibly annoying), since it feels like a balance that satisfies the requirements of unit testing while maintaining pace

Reply:
Status: this component, get a more reasonable 45 line snapshot as is... 
- toJson `{mode: 'deep'}` keeps it the same 45'ish lines 
- removing `.render()` balloons it out to double'ish the lines, 
- toJson seems to balloon it to the same double sized as removing `.render()`
- toJson `{mode: 'shallow'}` shrinks it for sure, but appears to not contain enough to be helpful